src/d3a/d3a_core/export.py

```diff
@@ -61,6 +64,24 @@ def __init__(self, root_area: Area, path: str, subdir: str):
             os.makedirs(self.plot_dir)
 
         self.export()
+        self.export_json_data(self.directory)
+
+    def export_json_data(self, directory: dir):
+        json_dir = os.path.join(directory, "aggregated_results")
+        mkdir_from_str(json_dir)
+        kpi_file = os.path.join(json_dir, "KPI")
+        with open(kpi_file, 'w') as outfile:
+            cep_share = self.kpi.cep_energy / self.kpi.total_energy
+            self.kpi.performance_index["global-non-renewable-energy-share"] = cep_share
```

Comment:
Trivial, but why not moving this calculation inside the kpi as well? cep_energy, total_energy and performance_index are all members of the kpi class, therefore this calculation could be moved inside the class